src/shim/util/configHelper.ts

```diff
@@ -0,0 +1,81 @@
+import { HttpInstrumentationConfig } from "@opentelemetry/instrumentation-http";
+import { ApplicationInsightsOptions, ExtendedMetricType } from "../../types";
+import * as http from "http";
+
+class ConfigHelper {
```

Comment:
It feels you don't need a class here, everything is static and there are no attributes in it, just export the methods directly instead like we do in some other utils